quipucords/tests/utils/raw_facts_generator.py

```diff
@@ -132,7 +138,9 @@ def _vcenter_raw_facts():
         "vm.mac_addresses": [_faker.mac_address()],
         "vm.memory_size": None,
         "vm.name": _faker.slug(),
-        "vm.os": None,
+        "vm.os": _faker.random_element(["red hat enterprise linux", "rhel"])
+        if _faker.pybool()
+        else None,
```

Comment:
this formatting is... weird 🤔

Reply:
I think black would use parenthesis here and add indentation.

Reply:
Yeah… you can thank `ruff format` for that one. 😕